torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -256,6 +257,22 @@ def maybe_mark_dynamic_helper(t: torch.Tensor, dims: set[int]):
         t._dynamo_weak_dynamic_indices = dims.copy()  # type: ignore[attr-defined]
 
 
+def _should_disable_saved_tensors_hooks():
+    if torch._dynamo.compiled_autograd.in_compiled_autograd_region:
```

Comment:
maybe add a comment that we just don't support compiled autograd right now (so people reading this know that it's not fundamental)